vault_contract/README.md

```diff
@@ -27,14 +27,16 @@ Represents a digitally signed statement made by an issuer about a DID subject.
 | `id`                | `String`  | Unique identifier (e.g., `t5iwuct2njbbcdu2nfwr32ib`).                          |
 | `data`              | `String`  | VC data encrypted utilizing a key agreement algorithm for heightened security. |
 | `issuance_contract` | `Address` | Smart contract address responsible for verifiable credential issuance.         |
+| `issuer_did`        | `String`  | Issuer DID issuing the verifiable credential.                                  |
```

Comment:
IMO "Issuer DID issuing" is redundant. What about: `DID of the issuer of the verifiable credential.`?